packages/hw-app-btc/src/Btc.js

```diff
@@ -32,6 +32,20 @@ export default class Btc {
     transport.setScrambleKey("BTC");
   }
 
+  toHexDigit = number => {
+    var digits = "0123456789abcdef";
+    return digits.charAt(number >> 4) + digits.charAt(number & 0x0f);
+  };
+
+  toHexInt = number => {
+    return (
+      this.toHexDigit((number >> 24) & 0xff) +
+      this.toHexDigit((number >> 16) & 0xff) +
+      this.toHexDigit((number >> 8) & 0xff) +
+      this.toHexDigit(number & 0xff)
+    );
+  };
```

Comment:
can these 2 be implemented using Buffer instead? 

something like `buffer=Buffer.alloc(4); buffer.writeUInt32LE(number, 0); return buffer.toString("hex")` (not tested) ?

Reply:
Yes, I was just transcribing the code (not trying to improve it right away). I'll fix it.